Sources/KIF/Additions/UIView-Debugging.m

```diff
@@ -136,6 +136,9 @@ - (void)printDatePickerState {
         case UIDatePickerModeCountDownTimer:
             printf(" UIDatePickerModeCountDownTimer");
             break;
+        case UIDatePickerModeYearAndMonth:
```

Comment:
This silences an error

Reply:
This needs to be set for the spec to validate on new xcode, but also needs to be excluded for the tests to pass on older versions. fun